lightning/src/ln/channel.rs

```diff
@@ -5191,7 +5193,8 @@ impl<Signer: Sign> Writeable for Channel<Signer> {
 		self.config.forwarding_fee_proportional_millionths.write(writer)?;
 		self.config.cltv_expiry_delta.write(writer)?;
 		self.config.announced_channel.write(writer)?;
-		self.config.commit_upfront_shutdown_pubkey.write(writer)?;
+		// self.config.commit_upfront_shutdown_pubkey.write(writer)?;
```

Comment:
Yep, this line should just be removed - we can't change the bytes written here for backwards/forwards compatibility - we currently write one byte to indicate the status of `commit_upfront_shutdown_pubkey`, old versions expect to read exactly one byte for it, and current versions need to read exactly one byte for it to read content written by old versions.